sickbeard/server/web/core/error_logs.py

```diff
@@ -1,172 +1,177 @@
 # coding=utf-8
+"""Route to error logs web page."""
 
 from __future__ import unicode_literals
 
 import io
 import os
 import re
-from tornado.routes import route
+
+from mako.filters import html_escape
 import sickbeard
 from sickbeard import (
     classes, logger, ui,
 )
+from sickbeard.server.web.core.base import PageTemplate, WebRoot
 from sickrage.helper.encoding import ek
-from sickbeard.server.web.core.base import WebRoot, PageTemplate
+from tornado.routes import route
+
+
+# log regular expression: 2016-08-06 15:58:34 ERROR    DAILYSEARCHER :: [d4ea5af] Exception generated in thread DAILYSEARCHER
+log_re = re.compile(r'^(?P<year>\d{4})-(?P<month>\d{2})-(?P<day>\d{2})\s+'
+                    r'(?P<hour>\d{2}):(?P<minute>\d{2}):(?P<second>\d{2})\s+'
+                    r'(?P<log_level>[A-Z]+)\s+(?P<log_name>.+?)\s+::\s+(?P<log_message>.*)$')
+
```

Comment:
When the error is in the providers, the name of the  provider appear in the log message too

Also there is the numbers next to thread name like BACKLOGSEARCH-7385828